README.md

```diff
@@ -2,7 +2,7 @@
 
 # **Jessica Lee** (*preferred*: Jess)
 
-![This is me!](https://www.facebook.com/photo.php?fbid=1121862651210083&set=a.105723912823967&type=3&theater)
+![This is me!](/Users/JessicaLee/Desktop/Jess.jpg)
```

Comment:
Hi Jess! I am unable to view the photo. I am having this problem too!

Reply:
okay I think I figured it out:
What I did was to go to my forked hw01 repo, where I uploaded the photo of myself by clicking **Upload Files** which is near the Clone or download button. Once uploaded, it should show up as a file under README.md. I right clicked on the file name and copied the link address. I added that URL address in my README.md and I think the photo should display now